schema/artist/index.js

```diff
@@ -1,5 +1,7 @@
 import _ from 'lodash';
 import Image from '../image';
+import ArtworkSorts from '../sorts/artwork_sorts'
```

Comment:
Yea I was thinking depending on how some of the stuff shakes out if we start to put types into their own folders it would make sense to move sorts into there too.